Dockerfile

```diff
@@ -1,8 +1,16 @@
-FROM mas.ops.maap-project.org/root/jupyter-image/vanilla:develop
+FROM continuumio/miniconda3:4.7.12
```

Comment:
this is the public docker image that the ADE/DPS image is based on. We could probably use a python image instead instead since we're not using conda, but this is only ~150MB larger than that image, and has the same python installed, etc, so it's convenient to use.